mezzanine/core/static/mezzanine/js/admin/content_typed.js

```diff
@@ -0,0 +1,28 @@
+jQuery(function($) {
+    var toggleID = function(id) {
+        // Add or remove the page ID from the cookie IDs string.
+        var index = $.inArray(id, ids.split(','));
+        if (index == -1) {
+            if (ids) {ids += ',';}
+            ids += id;
+        }
+        document.cookie = cookie + '=' + ids + '; path=/';
+    };
+
```

Comment:
Cool. After first scanning the code I thought the ordering of the two events was important, but looking at it now I see it isn't. Not so tricky then. :-)